week3/homework/src/index.js

```diff
@@ -1,3 +1,56 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const express = require('express');
+const bodyParser = require('body-parser');
+const {
+    addItem,
+    removeItem,
+    listItems,
+    clearList,
+    updateItem,
+    markAsDone,
+    markAsNotDone
+} = require('./services');
+
+const app = express();
+app.use(bodyParser.json());
+
+// add new item to todo
+app.post('/add', (request, response) => {
```

Comment:
Since we are using the HTTP methods (`post`, `delete`, `get`, `put`) you don't really need the named endpoints: the post method is for creating a new item, so you could just `post` to `/todos` to create a new todo, or make a `get` request to `todos` to get all todos and make a `get` request to `todos/:id` to get a specific todo.

Reply:
Thanks @J05J4  for your kind review 👍 :)